src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -24,13 +27,14 @@ test('testing button text updates from ad params', () => {
     };
     testMap.onInit(eventData);
 
+    const buttonLabel = "Place";
     const startData = {
         messageId: 1,
         sessionId: "test-session-id",
         timestamp: 0,
         type: "SIMID:Player:startCreative",  
     }
```

Comment:
This startData object seems to be repeated in another test so maybe consider declaring it a top level const for reuse

Reply:
Since the eventData object is different in every test and therefore I declare it in the individual tests, I felt that having this startData object being declared as a top level const seems inconsistent. Since I only use this object twice, I didn't feel that it was a storage issue, but instead easier to read. Let me know your thoughts!

Reply:
Okay, sounds good!